Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.12-alpine as builder
+
+RUN apk add --no-cache make curl git gcc musl-dev linux-headers
+
+ADD . /go/src/github.com/smartcontractkit/bea-adapter
+RUN cd /go/src/github.com/smartcontractkit/bea-adapter && GO111MODULE=on GOOS=linux GOARCH=amd64 go build -o cl-bea
```

Comment:
OS and Arch should not be necessary to define explicitly in this case. Go modules can be enabled with `ENV GO111MODULE=on`

Reply:
I'll change this, thx